packages/docs/community/release-process/smart-contracts.md

```diff
@@ -34,26 +29,53 @@ Ongoing smart contract development is done on the `master` branch.
 ### Github tags
 All release branches should be tagged as such, e.g. `celo-contracts-N`. Each should include a summary of the release contents.
 
-## Build process
-A new release can be built by running the following script:
+### Contracts
+Every deployed smart contract has its current version number as a constant which is publicly accessible via the `getVersion()` function, which returns the storage, major, minor, and patch version. Version number is encoded in the Solidity source and updated as part of code changes.
+
+Contracts deployed to a live network without the `getVersion()` function, such as the original set of core contracts, are to be considered version `1.0.0.0`.
+
+## Build and Release Process
+Using the following scripts, new contracts can be built and deployed alongside a corresponding on-chain governance proposal. Run the following scripts from the branch with the new release contracts:
 
 ```bash
-yarn run release-contracts -d $CURRENTLY_RELEASED_BRANCH -f $FROM_ADDRESS -n $NETWORK
-```
+## Script 1 ##
+# verify the current contract bytecode on the network matches the contracts currently on the RC1 branch
+yarn truffle exec --network $NETWORK ./scripts/truffle/verify-bytecode.js --build_artifacts build/rc1 --before_release_1
```

Comment:
Can we simplify this command? e.g. `yarn verify-bytecode -n rc1`?

Reply:
Separately can we `s/rc1/mainnet`?

Reply:
I think this is not in the scope of this PR but something that @yorhodes may be working on